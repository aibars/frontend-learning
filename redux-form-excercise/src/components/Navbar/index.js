import React from 'react';

const Navbar = () => (
  <div className="navbar">
    <span className="navbar__label">Simple Website Builder</span>
  </div>
);

export default Navbar;
