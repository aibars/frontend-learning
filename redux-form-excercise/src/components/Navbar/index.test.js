import React from 'react';
import { shallow } from 'enzyme';

import Navbar from '.';

describe('Navbar', () => {
  it('works ok', () => {
    const component = shallow(<Navbar />);

    expect(component).toMatchSnapshot();
  });
});
