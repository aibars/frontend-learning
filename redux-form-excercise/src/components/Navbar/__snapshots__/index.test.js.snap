// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar works ok 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <span
        className="navbar__label"
      >
        Simple Website Builder
      </span>,
      "className": "navbar",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Simple Website Builder",
        "className": "navbar__label",
      },
      "ref": null,
      "rendered": "Simple Website Builder",
      "type": "span",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <span
          className="navbar__label"
        >
          Simple Website Builder
        </span>,
        "className": "navbar",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Simple Website Builder",
          "className": "navbar__label",
        },
        "ref": null,
        "rendered": "Simple Website Builder",
        "type": "span",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
